skin.css

```diff
@@ -2,12 +2,82 @@
     background-color: #ddd;
 }
 
-button.btn {
+.btn {
+  background: rgba(255, 255, 255, 0);
   border-radius: 3px;
   border-style: solid;
   border-width: 1px;
+  font-size: 18px;
+  color: #417cce;
   font-weight: normal;
   line-height: 1.15;
   text-align: center;
   vertical-align: middle;
-}
\ No newline at end of file
+  padding: 14px 60px;
+  margin-left: 5px;
+  cursor: pointer;
+}
+
+button.btn--primary{
+  background: #0654ba;
+  border-color: #0654ba;
+  color: #fff;
+}
+
+button.btn--secondary{
+  background: #f7f6f5;
+  color: #797979;
+  border-color: #b2b2b2;
+}
+
+button.btn--large{
+  padding: 20px 105px;
+  font-size: 20px;
+}
+
+button.btn--extra-large{
+  padding: 25px 145px;
+  font-size: 23px;
+}
+
+
+/* Nav */
+
+.primary{
+  background: #0654ba;
+  border-color: #0654ba;
+  color: #fff;
+}
+
+a{

```

Comment:
In general it's a bad idea to style all elements of the same type to look the same. It becomes hard to change that if you want to make a special case look different.

This is especially true when you're doing things like `text-decoration: none`. Generally you want links to look like links, as people will understand what they do easier.

Reply:
Thank you Ali for your feedback I have replaced the a tag by nav-link class.